storage/core/src/main/java/com/hortonworks/streamline/storage/impl/jdbc/JdbcStorageManager.java

```diff
@@ -233,6 +234,9 @@ public void init(Map<String, Object> properties) {
             case "mysql":
```

Comment:
Ok. In that case there was some other place in code where mysql was removed as part of changeset of this PR.  I think it was an "if" check which enforced that only postgre is supported. Not sure if we need to change that but can you can double check we have it right to support mysql?